### PR TITLE
Show completions in the UI. r=vporof. Fixes Issue #200

### PR DESCRIPTION
**Pull request by bgrins**
r? @victorporof

### CODE REVIEW

File: app/ui/browser/views/navbar/location.jsx [hunk 1]
```diff
@@ -17,6 +17,15 @@ import Btn from './btn';
 
 import { fixURL, getCurrentWebView } from '../../browser-util';
 
+// Pref off completions for now since the view doesn't do anything besides
+// show them.
+const SHOW_COMPLETIONS = false;
```
Comment (bgrins) [inline app/ui/browser/views/navbar/location.jsx]: If you want to try it locally you'd need to flip this bool

File: app/ui/browser/views/navbar/location.jsx [hunk 2]
```diff
@@ -28,6 +37,18 @@ const LOCATION_BAR_STYLE = Style.registerStyle({
   borderRadius: '2px',
 });
 
+const LOCATION_BAR_AUTOCOMPLETE_STYLE = Style.registerStyle({
+  position: 'absolute',
```
Comment (victorporof) [inline app/ui/browser/views/navbar/location.jsx]: This makes it not look like the mockup. The code would also be simpler if we just made the autocompletion list part of the chrome, and not some floating thing.

https://people.mozilla.org/%7Eshorlander/stuff/images/fawkes-explorations-06-05-search.png

Comment (bgrins) [inline app/ui/browser/views/navbar/location.jsx]: I have lots of unanswered questions about that mock (like what happens with overflow), and trying to match it is proving tricky.  Are you OK with landing with the popup mode and then do styling updates in follow ups?

Comment (victorporof) [inline app/ui/browser/views/navbar/location.jsx]: As discussed on Slack, ok.

**Comment by bgrins** [comment]
This was merged

**Pull request closed without merging**